packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeature.java

```diff
@@ -72,9 +74,14 @@ public void updateBuilder(CaptureRequest.Builder requestBuilder) {
     if (!checkIsSupported()) {
       return;
     }
-    requestBuilder.set(
-        CaptureRequest.CONTROL_AE_REGIONS,
-        exposureRectangle == null ? null : new MeteringRectangle[] {exposureRectangle});
+    if (defaultExposureRectangle == null) {
+      defaultExposureRectangle = requestBuilder.get(CaptureRequest.CONTROL_AE_REGIONS);
```

Comment:
If I understand correctly, this method is to update the value for `CaptureRequest.CONTROL_AE_REGIONS`. If this is already set, i.e. `defaultExposureRectangle` is non-null, and `exposureRectangle` is non-null, is there any reason to re-set it?

Reply:
Based on my understanding, this method specifically get the current metering areas that the device provide, if it set and the features is supported by the OEM. It's for later use if our metering builder returns null.

The reason we're getting the default metering value is because some devices misbehave when we set the metering values as null. Resulting in black viewfinder or outright crash.

Reply:
Will also add test later